### DISCUSSION TITLE
Asset alerts query

### DISCUSSION THREAD

**Discussion by andyhill2023** [snipe/snipe-it#13173] [Q&A]
Hi,

I'm looking at snipe-it as a possibility for my organisation.

One of the criteria is that we PAT test our electrical equipment every two years. Is there an option to setup an email alert when a device is due for PAT testing?

Thanks in advance.

**Comment by qay21** [comment]
SnipeIT uses the concept of "audit" for assets : an audit is basically checking where is an asset, and in which condition. A free text field is also included in order to add notes.

Audits can be planned, either to a specific date or to a certain amount of time after the last registered audit.

There is also a "Maintenance" section for each asset, in which you can add entries of different types. One of the available type is "PAT Test". However, maintenances can only be "declared", not planned, as far as I know.

Feel free to try out different things using the public demo : https://snipeitapp.com/demo
